### PR TITLE
Redesign of ResourceDetails and PodTemplateTable

### PR DESCRIPTION
**Pull request by chriskari**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Review by akucharska — APPROVED** [review]

File: public/i18n/en.yaml [hunk 1]
```diff
@@ -918,6 +919,7 @@ namespaces:
     memory-quotas: Memory Quotas
   description: <0>Namespace</0> helps to maintain and divide resources in a cluster.
   name_singular: Namespace
+  namespace-details: Namespace Details
   namespaces-overview: Namespaces Overview
   no-namespaces-found: No Namespaces fetched
```
[the remark this answers is not in the document]
Comment (chriskari) [reply public/i18n/en.yaml]: Since this translation is not used anymore I just removed it completely

**Review by IwonaLanger — APPROVED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]
Agata approved changes, so approve on behalf after fighting with conflicts and CLA.

**Pull request merged**